app/Http/Controllers/RedirectsController.php

```diff
@@ -81,7 +82,13 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        $redirect = Redirect::find($id);
```

Comment:
Not at all necessary for merge, but you should be able to inject this from the controller [with a type-hinted parameter](https://laravel.com/docs/5.2/routing#route-model-binding).

```diff
-   public function update(Request $request, $id)
+   public function update(Redirect $redirect, Request $request)
   {
-    $redirect = Redirect::find($id);

```